Aleph1.Skeletons.WebAPI/Aleph1.Skeletons.WebAPI.Pages/src/resources/helpers/space-or-punctuation.ts

```diff
@@ -0,0 +1,5 @@
+/**
```

Comment:
It is used to help omit special characters or punctuation marks from text when indexing for in-memory search engines such as [MiniSearch](https://github.com/lucaong/minisearch).

It might not fit everyone so can be deleted.